app/frontend/src/cljs/teet/project/project_view.cljs

```diff
@@ -142,6 +137,64 @@
       (when (:geometry-range? map-settings)
         [search-area-view/feature-search-area e! app project related-entity-type])]]))
 
+(defn- project-timeline [{:thk.project/keys [estimated-start-date estimated-end-date lifecycles]
+                          :as project}]
+  (r/with-let [show-in-modal? (r/atom false)]
+    (when (and estimated-start-date estimated-end-date)
+      (let [tr* (tr-tree [:enum])
+            project-name (project-model/get-column project :thk.project/project-name)
+            timeline-component [timeline/timeline {:start-date estimated-start-date
```

Comment:
As discussed with Joni, if we get back to the timeline component at some point, this should be refactored so that the component takes the project as an arguments and does all the necessary wrangling within the component.